fiftyone/utils/eval/coco.py

```
@@ -755,13 +795,15 @@ def _compute_pr_curves(samples, config, classes=None, progress=None):
                 for ri, pi in enumerate(inds):
                     q[ri] = pre[pi]
                     t[ri] = confs[pi]
+
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)